src/readmeoss/agent/readmeoss.php

```diff
@@ -141,11 +141,38 @@ function processUploadId($uploadId)
     }
 
     $contents = array('licensesMain'=>$licenseStmtsMain, 'licenses'=>$licenseStmts, 'copyrights'=>$copyrightStmts, 'licenseAcknowledgements' => $licenseAcknowledgements);
+    $contents = $this->identifiedGlobalLicenses($contents);
     $this->writeReport($contents, $uploadId);
 
     return true;
   }
 
+  /**
+    * @brief Copy identified global licenses
+    * @param[int,out] array $contents
+    * @return array $contents with identified global license path
+    */
+  function identifiedGlobalLicenses($contents)
```

Comment:
The function name is not very descriptive. And I do not understand its functionality from the name or from the description.